### ISSUE TITLE
Add functionality to complete lower-incomplete Bose-Einstein integral

### ISSUE THREAD

**Issue by jrsmith3** [jrsmith3/ibei#57]
# Overview
The lower-incomplete Bose-Einstein integral is straightforward to compute if functionality to compute the full Bose-Einstein integral and upper-incomplete Bose-Einstein integral have been implemented. The scope of this ticket is to add functionality to compute the lower-incomplete Bose-Einstein integral.

# Related issues
* Depends on #43.
* Depends on #68.

**Comment by jrsmith3** [comment]
I think this is a duplicate of #44.

**Comment by jrsmith3** [comment]
It is a duplicate. Closing.
